less/index.less

```diff
@@ -67,110 +74,244 @@ h1, h2, h3, h4, h5 {
 .container {
     width: 880px;
     margin: auto;
+    max-width: 880px;
 }
 
-/* Header Styles */
+// Header Styles
 
 header {
     width: 100%;
     display: flex;
-}
-
-header nav ul {
-    margin-top: 40px; 
-    width: 610px;
-    justify-content: space-between;
-    display: flex;
-}
+    nav{
+        .dropdown{
+            display:none;
+        }
+        ul {
+            margin-top: 40px; 
+            width: 610px;
+            justify-content: space-between;
+            display: flex;
+            
+            a {
+                text-decoration: none;
+                color: #b0b0b0;
+            }
 
-header nav ul li a {
-    text-decoration: none;
-    color: #b0b0b0;
+            a:hover {
+                text-decoration: underline;
+                color: black;
+            }
+        }
+        .logo {
+            align-self: flex-end;
+            margin: 28px 0 0 91px;
+            filter:invert(100%);
+        }
+    }
 }
 
-header nav ul li a:hover {
-    text-decoration: underline;
-    color: black;
-}
 
-header .logo {
-    align-self: flex-end;
-    margin: 28px 0 0 91px;
-}
-
-/* Call To Action (CTA) Styles */
+// Call To Action (CTA) Styles
 
 .cta {
     display: flex;
     margin-top: 72px;
     justify-content: flex-end;
-}
+    
+    .cta-text {
+        font-size: 72px;
+        text-align: center;
+        letter-spacing: 5px;
+        display: flex;
+        justify-content: center;
+        flex-flow: column;
+        align-items: center;
+        margin: 0 82px 0 0;
 
-.cta .cta-text {
-    font-size: 72px;
-    text-align: center;
-    letter-spacing: 5px;
-    display: flex;
-    justify-content: center;
-    flex-flow: column;
-    align-items: center;
-    margin: 0 82px 0 0;
-}
-
-.cta .cta-text button {
-    border: 1px double black;
-    font-size: 16px;
-    background: white;
-    width: 170px;
-    height: 32px;
-    margin-top: 10px;
-    cursor: pointer;
-}
-
-.cta .cta-text button:hover {
-    background: black;
-    color: white;
+        button {
+            border: 1px double black;
+            color:white;
+            font-size: 16px;
+            background: @dark-desktop;
+            width: 170px;
+            height: 32px;
+            margin-top: 10px;
+            cursor: pointer;
+        }
+        button:hover {
+            background: white;
+            color: black;
+         }
+    }
 }
 
-/* Main Content Styles */
+// Main Content Styles
 
 .main-content {
-   border-top: 2px solid black; 
-   border-bottom: 2px solid black;
+   border-top: 2px solid white; 
+   border-bottom: 2px solid white;
    margin: 75px 0 35px 0;
    padding: 35px 0;
    font-size: 16px;
    line-height: 18px;
-}
 
-.main-content .top-content {
-    display: flex;
-    justify-content: space-around;
-}
+    .top-content {
+        display: flex;
+        justify-content: space-around;
+    }
 
-.main-content .middle-img {
-    display: block;
-    margin: 30px auto;
-}
+    .middle-img {
```

Comment:
I would add `width: 100%` to this img so it conforms to the width of your device.